zzzz_modular_occulus/code/game/objects/items/weapons/traitor_weapons.dm

```diff
@@ -0,0 +1,231 @@
+/obj/item/twohanded/garrote // 12TC traitor item
+	name = "fiber wire"
+	desc = "A length of razor-thin wire with an elegant wooden handle on either end.<br>You suspect you'd have to be behind the target to use this weapon effectively."
+	icon = 'zzzz_Modular_occulus/icons/obj/weapons.dmi'
+	icon_state = "garrot_wrap"
+	w_class = ITEM_SIZE_TINY
+	var/mob/living/carbon/human/strangling
+	var/improvised = 0
+	var/garrote_time
+
+
+/mob/living/proc/grabbedby(mob/living/carbon/user, supress_message = FALSE)
+	if(user == src || anchored)
+		return 0
+	if(!(status_flags & CANPUSH))
+		return 0
+
+	for(var/obj/item/weapon/grab/G in grabbed_by)
+		if(G.assailant == user)
+			to_chat(user, "<span class='notice'>You already grabbed [src].</span>")
+			return
+
+
+	var/obj/item/weapon/grab/G = new /obj/item/weapon/grab(user, src)
+	if(buckled)
+		to_chat(user, "<span class='notice'>You cannot grab [src]; they are buckled in!</span>")
+	if(!G)	//the grab will delete itself in New if src is anchored
+		return 0
+	user.put_in_active_hand(G)
+	G.synch()
+	LAssailant = user
+
+	playsound(src.loc, 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
+	/*if(user.dir == src.dir)
+		G.state = GRAB_AGGRESSIVE
+		G.last_upgrade = world.time
+		if(!supress_message)
+			visible_message("<span class='warning'>[user] has grabbed [src] from behind!</span>")
+	else*///This is an example of how you can make special types of grabs simply based on direction.
+	if(!supress_message)
+		visible_message("<span class='warning'>[user] has grabbed [src] passively!</span>")
+
+	return G
+
+/obj/item/twohanded/garrote/Destroy()
+	strangling = null
+	return ..()
+
+/obj/item/twohanded/garrote/update_icon()
+	if(strangling) // If we're strangling someone we want our icon to stay wielded
+		icon_state = "garrot_unwrap"
+		return
+
+	icon_state = "garrot_[wielded ? "un" : ""]wrap"
+
+/obj/item/twohanded/garrote/improvised // Made via tablecrafting
+	name = "garrote"
+	desc = "A length of cable with a shoddily-carved wooden handle tied to either end.<br>You suspect you'd have to be behind the target to use this weapon effectively."
+	icon_state = "garrot_I_wrap"
+	improvised = 1
+
+/obj/item/twohanded/garrote/improvised/update_icon()
+	if(strangling)
+		icon_state = "garrot_I_unwrap"
+		return
+
+	icon_state = "garrot_I_[wielded ? "un" : ""]wrap"
+
+/obj/item/twohanded/garrote/wield(mob/living/carbon/user)
+	if(strangling)
+		user.visible_message("<span class='info'>[user] removes the [src] from [strangling]'s neck.</span>", \
+				"<span class='warning'>You remove the [src] from [strangling]'s neck.</span>")
+
+		strangling = null
+		update_icon()
+		STOP_PROCESSING(SSobj, src)
+
+	else
+		..()
+
+/obj/item/twohanded/garrote/attack(mob/living/carbon/M as mob, mob/user as mob)
```

Comment:
If this can only be used on humans, why not typecast the variable here?

Reply:
Monkeys subtype of human, etc.

Reply:
As shadow said.

Reply:
Typecasted the variable.